src/Comment/Comment.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+import './Comment.scss'
+
+export class Comment extends Component  {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      comment : ''
+    }
+  }
+
+    updateValue = (event) => {
+      event.preventDefault();
+      this.setState({[event.target.name]: event.target.value});
+    }
+
+    postComment = () => {
+        if(!this.state.comment || !this.props.name){
+            return false
```

Comment:
Did you want to display some kind of message if a user doesn't have the correct credentials? I think from what I read on the rubric was a user should be notified in something was missing.